polaris-ui/src/app/features/cases/hooks/use-case-details-state/useCaseDetailsState.ts

```diff
@@ -345,7 +325,7 @@ export const useCaseDetailsState = (
       });
       trackEvent("Close Document", { documentId: documentId });
     },
-    [dispatch]
+    [dispatch, trackEvent]
```

Comment:
@renjithabby is this safe for me to add `trackEvent` in as a dependency?  The compiler complains that it is not there, but adding it may have unforeseen consequences, like rerunning the event too often?

Reply:
i think in this case it is fine since useCallback function  updates the functions, whenever the dependent property is changed. But it wont make any call itself. So i wont see any issues.